perma_web/perma/celery_tasks.py

```diff
@@ -1224,3 +1228,74 @@ def conditionally_queue_internet_archive_uploads_for_date_range(start_date_strin
 
     else:
         logger.info("Skipped the queuing of file upload tasks: max tasks already in progress.")
+
+
+# WACZ CONVERSION
+
+def format_size(bytes_val):
+    """
+    Converts bytes to KB and MB
+    """
+    kilobytes = bytes_val / 1024
+
+    if kilobytes < 1:
+        return f"{bytes_val} bytes"
+    elif kilobytes < 1024:
+        return f"{math.ceil(kilobytes)} KB"
+    else:
+        megabytes = kilobytes / 1024
+        return f"{megabytes:.2f} MB"
+
+
+def format_time(seconds_val):
+    """
+    Converts seconds to minutes
+    """
+    if seconds_val >= 60:
+        return f"{round(seconds_val / 60, 1)} minutes"
+    else:
+        return f"{math.ceil(seconds_val)} seconds"
+
+
+@shared_task
+def convert(input_path, output_folder, benchmark_log):
```

Comment:
Although we are still in the prototyping phase, I think it is probably worth giving more descriptive names to functions related to this task, given that all celery tasks live in the same scope.

Reply:
done!